### PR TITLE
Azure files Sitecore

### PR DESCRIPTION
**Pull request by jrudley**
In a comment on the pull request, include the following information and check off the list.

 - [X] Mini bio
  - 11 years IT engineering, focused on cloud automation and large scale implementations 
 - [X] Desired date of publishing
  - 03-23-2016
 - [x] Social media contact info n/a
 - [x] Redaction: (yes|no) Yes
  - Allow redaction, editing or structure changes by technical writer staff
 - [x] Comments: (yes|no) No
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

File: _posts/2016-03-23-Sitecore-Azure-Files.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+date: 2016-03-23 10:00
+comments: false
+author: Jimmy Rudley
+authorIsRacker: true
+published: true
+categories:
+- Devops
+---
+
+Azure file storage is a great SMB storage offering that I often see go unused for a simple centralized file storage share. A great feature is mounting the share as a mapped network drive on your local machine.
```
Comment (catlook) [inline _posts/2016-03-23-Sitecore-Azure-Files.md]: ... storage offering for a simple centralized file storage share that I often see go unused. A super feature is the ability to mount the share ...

File: _posts/2016-03-23-Sitecore-Azure-Files.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+
+<!-- more -->
+
+When issues happen in Sitecore, if it be a crash dump, logs need archived, you want to do a deployment, or you just need to share something with a developer, using an Azure file storage share is a great option. I will show you how to easily setup an Azure file share and archive the Sitecore log files automatically.
```
Comment (catlook) [inline _posts/2016-03-23-Sitecore-Azure-Files.md]: ... in Sitecore, whether there's been a crash dump, logs ...
... great option. Let me show you how ...

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-d2cc22df13/).

File: _posts/2016-03-23-Sitecore-Azure-Files.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+- Devops
+---
+
+Azure file storage is a great SMB storage offering that I often see go unused for a simple centralized file storage share. A great feature is mounting the share as a mapped network drive on your local machine.
+
+<!-- more -->
+
+When issues happen in Sitecore, if it be a crash dump, logs need archived, you want to do a deployment, or you just need to share something with a developer, using an Azure file storage share is a great option. I will show you how to easily setup an Azure file share and archive the Sitecore log files automatically. 
+
+Let's create the Azure resources required to set up a file share. I will use Azure PowerShell to provision my Azure resources.
```
Comment (catlook) [inline _posts/2016-03-23-Sitecore-Azure-Files.md]: ... up a file share, using Azure PowerShell to provision the Azure resources.

File: _posts/2016-03-23-Sitecore-Azure-Files.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+     if (Test-Path "$($mappedDrive.DeviceID)\$tempFolderPath.zip")
+     {
+        Remove-Item -Path "$sitecoreDataPath\logs\$tempFolderPath\" -Recurse 
+     }
+    }
+}
+
+```
+
+We can easily add this powershell script to a task scheduler job to archive our logs. I hope this starts to help generate new ideas on how we could utilize Azure file storage, especially with Sitecore.
```
Comment (catlook) [inline _posts/2016-03-23-Sitecore-Azure-Files.md]: ... I hope helps generate new ideas on how to use Azure file ...

[ Ed note:  Thanks for sharing!  Use any edits you like and discard he rest! - Cat ]

**Comment by jrudley** [comment]
Thanks, Cat! Made the suggested changes

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-1c3be90aa2/). :bow:

**Comment by kenperkins** [comment]
@jrudley I'd like to explicitly call out how awesome @smashwilson's work is to enable PR full site preview builds as commented by rackernexus. Let us know if you have any feedback about the preview build.

**Comment by jrudley** [comment]
You know, I already let my product team know of the new changes when I saw the preview. My exact words to them  "you guys will like the new pr preview, it's pretty cool" 

So, it's pretty cool :) nice work

**Comment by jrudley** [comment]
made a fix to a typo and removed a word that may cause confusion

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-4e9a503200/).

**Comment by jrudley** [comment]
So, do I need to do something during the preview to make the merge happen
to master? Not sure if the process changed on getting my content published?

Thanks

**Comment by smashwilson** [comment]
Nope, one of us needs to merge it in. Sorry for the delay, I've been putting out ops :fire:

**Comment by smashwilson** [comment]
It should be live shortly!

**Comment by jrudley** [comment]
ah, no worries. Just need this in place by Wednesday as we have a meeting
with E&Y in New York and want to demo some stuff.

Thanks!

**Pull request merged**